docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -25,7 +30,9 @@ image:images/uptime-set-up-heartbeat.png[]
 You can now create monitors directly in the {uptime-app}.
 To try out this beta functionality, you will need to enable Monitor Management.
 
-This approach works well if you want to configure and update monitors using a UI in your browser.
+This approach works well if you want to configure and update monitors using a
+UI in your browser or using a GitOps workflow.
```

Comment:
I think this may be a little confusing, if they are using the Uptime app suggested of using the UI to configure monitors (not project monitors via config files)

Reply:
That's fair. Would "Monitor management" be an appropriate alternative or do you think they need to presented as three distinct options?